### DISCUSSION TITLE
integrated coravel pro

### DISCUSSION THREAD

**Discussion by orangesoncom** [jamesmh/coravel#293] [Q&A] [answered]
I've been using Coravel for my console application and it's running fine, 
then I need a dashboard that can monitor or change the schedule through the dashboard like the one on Coravel Pro, 
so can the schedule I create in the console app be on the Coravel Pro dashboard? 
are there any instructions I can follow?

**Comment by jamesmh** [comment, marked as answer]
Hey,
 
With Coravel Pro you would have to take all the scheduled jobs that are already statically scheduled in your code (e.g. in `Program.cs`) and re-schedule them via Coravel Pro's UI (this is because the jobs scheduled from the UI are persisted).

There is an upcoming feature that will display any scheduled jobs scheduled via code (but immutable from the UI) but that currently isn't available.

Does that answer your question? Thanks!

**Comment by orangesoncom** [comment]
do you mean I have to rewrite my code (console app) into Coravel Pro, so no more console app, sorry for my English

**Comment by jamesmh** [comment]
Your existing code would work the same, but you would not see any metrics etc. for those jobs since they are not scheduled through the UI.

**Reply by orangesoncom** [reply]
ok got it, thanks for your kindly reply
